apps/table-frontend/src/app/app.css

```diff
@@ -1,132 +0,0 @@
-/*
```

Comment:
did you mean to remove all the css?

Reply:
Oh yeah, there was boilerplate css which was just from the initial template. But I kept the file for future use